pages/exercises/[lessonSlug].tsx

```diff
@@ -199,14 +204,16 @@ const ExerciseList = ({
         </div>
       </div>
       <div className={styles.exerciseList__container}>
-        {mockExercisePreviews.map((exercisePreview, i) => (
+        {exercises.map((exercise, i) => (
           <ExercisePreviewCard
             key={i}
-            moduleName={exercisePreview.moduleName}
-            state={exercisePreview.state}
-            problem={exercisePreview.problem}
+            moduleName={exercise.moduleName}
+            state={exercise.userAnswer === null ? 'NOT ANSWERED' : 'ANSWERED'}
```

Comment:
Right now there are only 2 states, but I think there should be a third state in the future called "ANSWERED INCORRECTLY" or something similar. Maybe we can make "ANSWERED INCORRECTLY" red and then "NOT ANSWERED" can be gray or some neutral color.